### DISCUSSION TITLE
Let `validate` return a string as error message

### DISCUSSION THREAD

**Discussion by pke** [react-hook-form/react-hook-form#3092] [Ideas] [answered]
Currently it seems like `validate` is the only rule that can not return a string to indicate its error message.

How about changing this to 

```
validate: (value) => value === 42 0 ? "Can't be 42" : true
```

**Comment by bluebill1049** [comment, marked as answer]
pretty sure validate can return string

**Reply by pke** [reply]
You are right! None of the examples does it though, but a peek into the source code helped to clear this up.

**Reply by bluebill1049** [reply]
<img width="1000" alt="Screen Shot 2020-10-06 at 7 06 27 pm" src="https://user-images.githubusercontent.com/10513364/95175326-0b299580-0807-11eb-8eb7-3097db47c869.png">

pretty sure we do, there is a radio button to toggle the error message at register
